include/copc-lib/io/writer.hpp

```diff
@@ -97,10 +107,37 @@ class Writer : public BaseIO
         throw std::runtime_error("No pages should be unloaded!");
     };
 
+    Writer() = default;
```

Comment:
Put this directly under protected:

Reply:
I forget, do we actually need a default constructor to make a subclass?

Reply:
We do yeah, so I'll move it up